file/cache.lisp

```diff
@@ -2,11 +2,15 @@
 ;;;; This code is licensed under the MIT license.
 
 (defpackage :mezzano.file-system-cache
-  (:use :cl)
+  (:use :cl :mezzano.disk)
```

Comment:
Avoid using packages (aside from cl), use a local nickname & explicit package prefix instead. This makes it clearer where symbols are coming from.